renderer/reducers/channels.js

```diff
@@ -838,6 +889,16 @@ const initialState = {
     { key: 'CLOSING_PENDING_CHANNELS', name: 'Closing' },
   ],
 
+  sort: 'OPEN_DATE',
+  sorters: [
+    { key: OPEN_DATE, name: 'Open date' },
```

Comment:
Names should probably be added in the components and made i18n aware

Reply:
This is actually consistent with how other dropdowns are handled. Like the filter above, or dropdown on the activity screen. I feel like it's outside of the scope of this PR and a dedicated PR should be done to tackle those other cases too and to have some sort of a consistent scheme to do i18n in dropdowns.

Reply:
Agreed. I've created https://github.com/LN-Zap/zap-desktop/issues/2073 to handle this